scripts/spack/configs/blueos_3_ppc64le_ib_p9/packages.yaml

```diff
@@ -134,7 +134,7 @@ packages:
     version: [10.0.0]
     buildable: false
     externals:
-    - spec: llvm+clang
+    - spec: llvm+clang@10.0.0
```

Comment:
The version above this is supposed to handle this.  Was this a problem?

Reply:
They serve different purposes - the first one tells Spack that it needs to be using LLVM 10, the second one tells Spack that the external LLVM installation is version 10.  The new concretizer requires that all externals have a version associated with it (hence all the changes).